com.discordapp.Discord.json

```
@@ -23,9 +23,13 @@
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--talk-name=com.canonical.AppMenu.Registrar",
         "--talk-name=com.canonical.indicator.application",
-        "--talk-name=com.canonical.Unity.LauncherEntry",
+        "--talk-name=com.canonical.Unity",
         "--own-name=org.kde.*",
-        "--env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons"
+        "--env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons",
+        /* Electron checks if it's running on Unity/KDE before using the UnityLauncherAPI. To make it work on other Desktops too, we need to pretend we are on KDE. */
```

It his tripping over this comment, maybe this works?

```suggestion
        /* Electron checks if it's running on Unity or KDE before using the UnityLauncherAPI. To make it work on other Desktops too, we need to pretend we are on KDE. */
```